daemon/graphdriver/driver.go

```diff
@@ -142,10 +142,36 @@ func GetDriver(name, home string, options []string) (Driver, error) {
 func New(root string, options []string) (driver Driver, err error) {
 	for _, name := range []string{os.Getenv("DOCKER_DRIVER"), DefaultDriver} {
 		if name != "" {
+			logrus.Infof("[graphdriver] trying provided driver %q", name) // so the logs show specified driver
 			return GetDriver(name, root, options)
 		}
 	}
 
+	// Guess for prior driver
```

Comment:
> But I feel there is a case needing to allow bouncing back and forth... 

But isn't that where the `-s` flag? If I am testing graph drivers, I can just restart the daemon with a different `-s` flag to toggle between graph drivers.

Bailing out with;

    -----------------------------------------------
    ERROR
    -----------------------------------------------

    The docker daemon failed to start because we were unable to determine 
    the graph driver you're using. (Multiple graphs were found).

    The following graphs were found;

    - overlay
    - aufs
    - devmapper

    Please specify which graph driver should be used by setting the '-s **drivername**' 
    option and restart the daemon to use the selected graph driver.

(Or similar) Doesn't sound too bad?